Adjust Keda docs according to the documentation requirements (#156)
kyma-project/keda-manager: docs/user/01-01-keda-module.md

> # Keda Module

Reviewer: This file looks like a main landing page for users?
Should it be renamed to`readme.md` so that it renders nicely in github when sb arrives via such link ?
`https://github.com/kyma-project/keda-manager/tree/main/docs/user`

## What is KEDA

Kubernetes-based Event Driven Autoscaler [(KEDA)](https://keda.sh/) is an autoscaler that allows you to scale easily your Kubernetes-based resources. You can scale your applications on the basis of data of your choice.

Keda supports a great number of scalers that help you manage your deployments. For the complete list, check the KEDA [Scalers](https://keda.sh/docs/scalers/) documentation.

For more information about KEDA features, see [KEDA documentation](https://keda.sh/docs).

## Keda module

> Keda module is a solution introduced in Kyma that allows you to install and manage KEDA on your Kubernetes cluster, using Keda Manager.

Reviewer: ```suggestion
Keda module is an extension to Kyma that allows you to install and manage KEDA on your Kubernetes cluster, using Keda Manager.
```

To learn how to enable and disable the Keda module, visit {LINK}.

## Keda Manager

Keda Manager helps you to install and manage KEDA on your cluster. It manages the lifecycle of KEDA based on the dedicated Keda custom resource (CR).

> ## User interface (UI)

Reviewer: This section is for contributors.
I propose moving it to a separate file "extending UI"
